config/config.go

```diff
@@ -165,6 +168,11 @@ func validateConfig(c *Config) error {
 			"'tls_key_file' and 'tls_cert_file'")
 	}
 
+	if c.TLSCaFile != "" && (c.TLSCertFile == "" || c.TLSKeyFile == "") {
+		return errors.New("When enabling mTLS one must specify a: 'tls_ca_file' " +
+			"as well as 'tls_cert_file', and 'tls_key_file'")
```

Comment:
We don't seem to use the term "mTLS" anywhere else, would it be better to use the term "client certificates" here?